### PR TITLE
Sscs deps update 2019 04

### PR DESCRIPTION
**Pull request by keefmarshall**
Please remove this line and everything above and fill the following sections:

### Change description ###

Updated SSCS dependencies and added new services

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: docs/microservices.json [hunk 1]
```diff
@@ -1659,28 +1659,145 @@
...
+                    "apis": []
+                },
                 {
                     "id": "rpe-service-auth-provider",
                     "hard": true,
                     "apis": []
                 },
                 {
-                    "id": "sscs-tya-frontend",
+                    "id": "cmc-pdf-service",
```
Comment (doncem) [inline docs/microservices.json]: this has finally converted to `rpe-pdf-service` but that's not part of this pr - needs a global update

**Comment by timja** [comment]
@keefmarshall are you done with this? / ready to merge?
